publishing-bot/configs/kubernetes-rules-configmap.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: publisher-rules
+data:
+  config: |
```

Comment:
The same question as above

Reply:
Answered here: https://github.com/kubernetes/k8s.io/pull/520#discussion_r368515333